intermediate-javascript-calorie-counter/index130.js

```diff
@@ -0,0 +1,24 @@
+document.getElementById('calorie-form').onsubmit = calculate;
+
+function calculate(e) {
+  e.preventDefault();
+
+  const total = Array.from(
+    document.getElementsByClassName('cal-control')
+  ).map();
+}
+/*
+Now we need to provide a function to `map()` that will be performed on each item of the array.
```

Comment:
I think this lesson can be split into two or maybe three lessons. The first can have the student write an empty longer version of the function (function(meal) { }), the second could have them return meal.value and log `total` to the console, and the third could have them refactor it into an arrow function.